src/main/resources/Ideas/IdeasLiveTableResults.xml

```diff
@@ -19,9 +19,5 @@
   <minorEdit>false</minorEdit>
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
-  <content>{{velocity output="false"}}
-$services.localization.use('document', 'Ideas.IdeasTranslations')
-{{/velocity}}
-  
-{{include document="XWiki.LiveTableResults" /}}</content>
+  <content>{{include document="XWiki.LiveTableResults" /}}</content>
```

Comment:
This page should be deleted altogether, as well as the line calling it in the Ideas.WebHome livetable.
By default XWiki.LiveTableResults is called, there's no need to have an extra document with just an include of it.